elm/ords/process.py

```diff
@@ -485,7 +486,7 @@ async def download_docs_for_county_with_logging(
         return doc
 
 
-async def download_doc_for_county(
+async def process_county(
```

Comment:
Yea, thanks for these re-names. Much better function name (20/20 hindsight lol)

Reply:
all good, little things like this will make it easier for me and reid to pick up and apply for other technologies.